drivers/spi/spi_ll_stm32.c

```diff
@@ -173,6 +344,9 @@ static void spi_stm32_shift_s(SPI_TypeDef *spi, struct spi_stm32_data *data)
  */
 static int spi_stm32_shift_frames(SPI_TypeDef *spi, struct spi_stm32_data *data)
 {
+#ifdef CONFIG_SPI_STM32_DMA
```

Comment:
This won't work when DMA and non DMA operate in same binary

Reply:
what is not supported is spi1 Tx with dma and spi1 Rx without dma or  spi1 Rx with dma and spi1 Tx without dma